tests/integration/test_fim/test_files/test_audit/test_audit.py

```diff
@@ -301,7 +301,14 @@ def get_audit_creation_time():
                 return proc.create_time()
         pytest.fail("Auditd is not running")
 
-    plugin_path = "/etc/audisp/plugins.d/af_wazuh.conf"
+    audit2_path = "/etc/audisp/plugins.d/af_wazuh.conf"
+    audit3_path = "/etc/audit/plugins.d/af_wazuh.conf"
+    if os.path.exists(audit2_path):
+        plugin_path = audit2_path
+    elif os.path.exists(audit3_path):
+        plugin_path = audit3_path
```

Comment:
```suggestion
    audisp_path = "/etc/audisp/plugins.d/af_wazuh.conf"
    audit_path = "/etc/audit/plugins.d/af_wazuh.conf"
    
    if os.path.exists(audisp_path):
        plugin_path = audisp_path
    elif os.path.exists(audit_path):
        plugin_path = audit_path
```

Reply:
Fixed in 9d9aae28f695f755d0b9dc34647087ad6729ad2e